src/main/scala/mesosphere/marathon/core/launcher/impl/TaskOpFactoryImpl.scala

```diff
@@ -129,7 +130,7 @@ class TaskOpFactoryImpl @Inject() (
     offer: Mesos.Offer,
     task: Task.Reserved,
     resourceMatch: Option[ResourceMatcher.ResourceMatch],
-    volumeMatch: Option[PersistentVolumeMatcher.VolumeMatch]): Option[TaskOp] = {
+    volumeMatch: Option[PersistentVolumeMatcher.VolumeResourceMatch]): Option[TaskOp] = {
```

Comment:
:+1: for also changing the name when the semantics change (or at least the context)